cmd/syncthing/main.go

```diff
@@ -634,12 +634,6 @@ func syncthingMain(runtimeOptions RuntimeOptions) {
 	errors := logger.NewRecorder(l, logger.LevelWarn, maxSystemErrors, 0)
 	systemLog := logger.NewRecorder(l, logger.LevelDebug, maxSystemLog, initialSystemLog)
 
-	// Event subscription for the API; must start early to catch the early
-	// events. The LocalChangeDetected event might overwhelm the event
-	// receiver in some situations so we will not subscribe to it here.
-	apiSub := events.NewBufferedSubscription(events.Default.Subscribe(events.AllEvents&^events.LocalChangeDetected&^events.RemoteChangeDetected), 1000)
-	diskSub := events.NewBufferedSubscription(events.Default.Subscribe(events.LocalChangeDetected|events.RemoteChangeDetected), 1000)
```

Comment:
Unless the reasoning in the above comment is no longer valid, wouldn't it make sense to keep these two subscriptions and add them to the new `eventSubs` map as default subscriptions? I suspect most API users will keep using the default call.
Also in general this approach differs significantly from the old behaviour, by not providing past events on the first call with a specific event type. On a second call, it will, which I think is confusing from a user perspective.